[QA] Fixes for v3.3.0

mishautkin opened the pull request:
### Description

- Fixes for running autotests for 3.3.0
- Enabled video recording, full page screenshots and tracing in playwright-report (for local execution only because of memory consumption)
- Added "Critical" labels to more tests

mishautkin committed:
Remove duplicate files (commit error)

mishautkin committed:
Update playwright-utils version

mishautkin committed:
Exclude payPalApi assertions from transaction tests

mishautkin committed:
Exclude payPalApi assertions from vaulting and subscription tests

mishautkin committed:
Remove extra assertion for my-account

mishautkin committed:
Soft assertions and fix renewal in pcp-api

mishautkin committed:
Fix assertion of vaulted PayPal button

mishautkin committed:
Fix mastercard for ACDC payments

mishautkin committed:
Update PCP-plugin version to 3.3.0

mishautkin committed:
Enable video recording and trace in playwright-report

mishautkin committed:
QA-legacy-ui: Enable video recording and trace in playwright-report

mishautkin committed:
QA-legacy-ui: Update PCP-plugin version to 3.3.0, update playwright-utils

mishautkin committed:
Mirot fixes in tests

mishautkin committed:
Add  @Critical tag to Default order tests

mishautkin committed:
Exclude fastlane from makeOrder on checkouts to separate method

puntope commented on tests/qa/tests/05-transactions/_test-scenarios/cart.scenario.ts:
```diff
@@ -42,11 +42,11 @@ export const transactionsOnCart = ( testsData: ShopOrder[] ) => {
+				// await payPalApi.assertPayment(
+				// 	orderJson.transaction_id,
+				// 	testData
+				// );
```
Lets completely remove old code if doesnt work

mishautkin replied:
It's been decided to skip these assertions recently. But we don't have other tests which would check it. That's why I only commented it out.

mishautkin replied:
Removed. We can add in future.

puntope reviewed (commented):
<img width="1531" height="171" alt="Screenshot 2025-11-13 at 16 49 30" src="https://github.com/user-attachments/assets/47fb4d2a-d59d-4573-b6d2-b3ddc74edc49" />

Some tests are failing for me

mishautkin committed:
Standardize getting parameters for transaction assertion

mishautkin committed:
Move page.visit() from makeOrder methods to test scenarios, remove meaningless methods.

mishautkin committed:
Standardize transaction tests

- Implement standard approach for loops in spec file instead of scenario
- Remane testData to testOrder in all transactions

mishautkin committed:
Move makeOrder method content to cart.scenario instead of Cart class

mishautkin committed:
Move makeOrder method content to classicCart.scenario instead of ClassicCart class

mishautkin committed:
Move makeOrder method content to vaulting-classic-cart.scenario instead of ClassicCart class

mishautkin committed:
Add missing checkout.visit()

mishautkin committed:
Remove applyCoupon methods from checkout classes

mishautkin committed:
optimize shipping selection on block checkout

mishautkin committed:
Fix some transaction test loops.

mishautkin committed:
Fix failing tests after debugging

mishautkin committed:
Update playwright-utils version

mishautkin committed:
Remove unused .zip

mishautkin committed:
Remove PCP plugin .zip from resources/files/ and add to .gitignore

mishautkin committed:
Standardize checkout flows

- Move makePayment method from POM classes to test scenarios
- Rename makeOrder to completeCheckoutDetails

mishautkin committed:
Remove commented assertions from transactions

mishautkin committed:
Lint fix

mishautkin committed:
Remove unused .zip plugins

mishautkin committed:
Remove installation of unused .zip plugins

mishautkin committed:
Fix typos

mishautkin committed:
Merge branch 'develop' of https://github.com/woocommerce/woocommerce-paypal-payments into qa-fix-330

mishautkin committed:
Enable video to fix fastlane tests

mishautkin committed:
Reduce CI test retries to 1

mishautkin committed:
Increase failed video record viewport to see element

mishautkin committed:
Increase failed video record viewport to see element

mishautkin committed:
Allow trace in CI, unify vuewport size

mishautkin committed:
Merge branch 'develop' of https://github.com/woocommerce/woocommerce-paypal-payments into qa-fix-330

mishautkin committed:
Remove Ryan's fastlane email from .env

mishautkin committed:
Fix emails for fastlane
